tests/saw/failure_tests/tls_missing_full_handshake.patch

```diff
@@ -8,7 +8,7 @@ index 14e6bb9f..e2e8869f 100644
  
 -    [NEGOTIATED | FULL_HANDSHAKE | OCSP_STATUS ] ={
```

Comment:
All the changes to the file are SERVER_SESSION_LOOKUP related. Since it's a patch file, it is a disaster to display. The FULL_HANDSHAKE lines that are being removed are part of the original patch file, and not the changes in the PR.